python/ray/_private/services.py

```diff
@@ -1070,16 +1071,19 @@ def _start_redis_instance(executable,
         command += (["--port", str(port), "--loglevel", "warning"])
         if listen_to_localhost_only:
             command += ["--bind", "127.0.0.1"]
+        fd, pidfile = tempfile.mkstemp()
```

Comment:
Actually, can we put this in the `/tmp/ray/session_latest` directory? 

That would mean calling `get_session_dir()` in `node.py` and then passing that result into this function.